dev/breeze/src/airflow_breeze/commands/setup_commands.py

```diff
@@ -508,7 +508,7 @@ def check_params(command: str, subcommand: str | None, command_dict: dict[str, A
             f"defined in rich click configuration."
         )
         get_console().print(f"[warning]Please add it to the `{command_path_config(command)}`.")
-        return False
+        return True
```

Comment:
It's kinda strange that for a function named `check_params` the return value of True means that something is wrong?

Reply:
Sure. Changed :)